src/index.ts

```diff
@@ -2,28 +2,51 @@ import { ModuleWithProviders, NgModule } from '@angular/core';
 
 // Modules
 import { NgAisBreadcrumbModule } from './breadcrumb/breadcrumb.module';
+export { NgAisBreadcrumbModule };
 import { NgAisClearRefinementsModule } from './clear-refinements/clear-refinements.module';
+export { NgAisClearRefinementsModule };
 import { NgAisCurrentRefinementsModule } from './current-refinements/current-refinements.module';
+export { NgAisCurrentRefinementsModule };
 import { NgAisHierarchicalMenuModule } from './hierarchical-menu/hierarchical-menu.module';
+export { NgAisHierarchicalMenuModule };
 import { NgAisHitsPerPageModule } from './hits-per-page/hits-per-page.module';
+export { NgAisHitsPerPageModule };
 import { NgAisHitsModule } from './hits/hits.module';
+export { NgAisHitsModule };
 import { NgAisInfiniteHitsModule } from './infinite-hits/infinite-hits.module';
+export { NgAisInfiniteHitsModule };
 import { NgAisInstantSearchModule } from './instantsearch/instantsearch.module';
+export { NgAisInstantSearchModule };
 import { NgAisMenuModule } from './menu/menu.module';
+export { NgAisMenuModule };
 import { NgAisNumericMenuModule } from './numeric-menu/numeric-menu.module';
+export { NgAisNumericMenuModule };
 import { NgAisNumericSelectorModule } from './numeric-selector/numeric-selector.module';
+export { NgAisNumericSelectorModule };
 import { NgAisPaginationModule } from './pagination/pagination.module';
+export { NgAisPaginationModule };
 import { NgAisRangeSliderModule } from './range-slider/range-slider.module';
+export { NgAisRangeSliderModule };
 import { NgAisRefinementListModule } from './refinement-list/refinement-list.module';
+export { NgAisRefinementListModule };
 import { NgAisSearchBoxModule } from './search-box/search-box.module';
+export { NgAisSearchBoxModule };
 import { NgAisSortByModule } from './sort-by/sort-by.module';
+export { NgAisSortByModule };
 import { NgAisRatingMenuModule } from './rating-menu/rating-menu.module';
+export { NgAisRatingMenuModule };
 import { NgAisStatsModule } from './stats/stats.module';
+export { NgAisStatsModule };
 import { NgAisToggleModule } from './toggle/toggle.module';
+export { NgAisToggleModule };
 import { NgAisHighlightModule } from './highlight/highlight.module';
+export { NgAisHighlightModule };
 import { NgAisRangeInputModule } from './range-input/range-input.module';
+export { NgAisRangeInputModule };
 import { NgAisPanelModule } from './panel/panel.module';
+export { NgAisPanelModule };
 import { NgAisConfigureModule } from './configure/configure.module';
+export { NgAisConfigureModule };
```

Comment:
you can write this as: 

```js
export { NgAisConfigureModule } from './configure/configure.module';
```

I think?

Reply:
I thought so too but it didn't seem to work.
I tried it out on a more [simple typescript boilerplate](https://codesandbox.io/s/n72q2p4q20) 
Since it's not import statement I guess it makes sense that it's doesn't introduces new modules to the current scope